tests/apitest/apitest.csproj

```diff
@@ -24,7 +24,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
-    <EnableCodeSigning>false</EnableCodeSigning>
+    <EnableCodeSigning>true</EnableCodeSigning>
```

Comment:
Was this an intentional change? Just asking for a friend :P

Reply:
Yes, otherwise the permission keys in the Info.plist don't work.